### DISCUSSION TITLE
simplelogin.com and simplelogin.fr domains lost

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1582] [Q&A]
Hello!
The ability to create aliases on the domains simplelogin.com and simplelogin.fr has disappeared. This problem is observed both in the IOS application and in the web version of the service. Fix it please!

Screenshots are attached:
[Screenshot 1](https://user-images.githubusercontent.com/126008136/220421548-220e5df3-5df1-4b60-ad44-f4f23bba2bca.jpeg)
[Screenshot 2](https://user-images.githubusercontent.com/126008136/220421571-aee645cc-a447-4f5c-9655-c376feeb55a9.jpeg)

**Comment by nguyenkims** [comment]
Hi this is expected. These are part of the domain changes we've done recently.

**Comment by nguyenkims** [comment]
This is duplicated to this discussion https://github.com/simple-login/app/discussions/1589, please continue the discussion there.
